t/bad_subclass.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env perl
+
+use lib 'lib', 't/lib';
```

Comment:
There's no need to use `lib` via `use lib`. That was in some other tests for some reason but I've now removed it in master.